### ISSUE TITLE
Add MHD results once Trixi's new handling of nonconservative terms is merged

### ISSUE THREAD

**Issue by ranocha** [trixi-framework/paper-2021-juliacon#3]
The EC flux should be `flux_hindenlang_gassner` (FLOGOR in FLUXO) with nonconservative terms given by `flux_nonconservative_powell`. This should be done once https://github.com/trixi-framework/Trixi.jl/pull/657 is merged into `main`.
